svelte/components/swarm/NftMintSwarm.svelte

```diff
@@ -0,0 +1,485 @@
+  let chainId: number;
+  let account: string;
+  let address: string;
+  let readableAddress: Readable<string>;
+
+  let files: FileList;
+  let file: File;
+  let image: string;
+  let nftTitle: string = "";
+  let nftDescription: string = "";
+  /////////////////////////////////////////////////
+  let swarm: string;
+  let swarmJson: string;
+  let minting: number;
+  let mintingTxResp: TransactionResponse;
+  let mintedNft: NftType;
+  let mintingError: string;
+  /////////////////////////////////////////////////
+  let open = false;
+
+  $: mintedNft && open === false && handleResetAfterMint();
+  const handleResetAfterMint = () => {
+    if (!src) {
+      files = null;
+      file = null;
+      image = null;
+      nftTitle = null;
+      mintReset();
+    }
+  };
+
+  const openSwarmMintModal = () => {
+    open = true;
+  };
+
+  const sell = (e: Event): void => {
+    e.preventDefault();
+    location.href = nftOpenSeaUrl($metamaskChainId, mintedNft);
+  };
+
+  const view = (e: Event): void => {
+    e.preventDefault();
+    location.href = nftGetImageLink(mintedNft);
+  };
+  /////////////////////////////////////////////////
+  // ON network or account change
+  $: $metamaskChainId && $metamaskSigner && handleChange().catch(console.error);
+  const handleChange = async () => {
+    chainId = $metamaskChainId;
+
+    account = await $metamaskSigner.getAddress();
+    console.log("handleChange", $metamaskChainId, account);
+
+    if (src) {
+      readableAddress = collectionStore.getDefaultSubStore($metamaskChainId, true, account);
+      address = $readableAddress;
+      console.log("handleChange ~ address", $readableAddress);
+    }
+  };
+
+  /////////////////////////////////////////////////
+  // ON Wordpress get file & nftTitle & image
+  $: src !== "" && handleWpFile().catch(console.error);
+  const handleWpFile = async (): Promise<void> => {
+    const blob = await fetch(src).then((r) => r.blob());
+    file = new File([blob], alt, { type: blob.type });
+
+    if (file) {
+      let reader = new FileReader();
+      reader.readAsDataURL(file);
+      nftTitle = file.name;
+      reader.onload = (e) => {
+        image = e.target.result.toString();
+      };
+    }
+  };
+
+  /////////////////////////////////////////////////
+  // ON Dapp (modal) AFTER upload get file & nftTitle & image to DISPLAY {image}
+  const fileload = () => {
+    mintReset();
+    file = null;
+
+    if (files) {
+      let reader = new FileReader();
+      reader.readAsDataURL(files[0]);
+      nftTitle = nftTitle || files[0].name;
+      nftDescription = nftDescription || files[0].name;
+      reader.onload = (e) => {
+        image = e.target.result.toString();
+      };
+
+      file = files[0];
+    }
+  };
+
+  /////////////////////////////////////////////////
+  const mintReset = (): void => {
+    swarm = null;
+    swarmJson = null;
+    minting = 0;
+    mintingTxResp = null;
+    mintedNft = null;
+    mintingError = null;
+  };
+
```

Comment:
this file should be same as NftMint, apart nftMint1 and nftMint2

Reply:
wip